test/index.js

```diff
@@ -563,69 +508,64 @@ describe('Mongo', () => {
 
             const mongo = new Mongo(options);
 
-            mongo.start((err) => {
-
+            try {
+                await mongo.start();
+            }
+            catch (err) {
                 expect(err).to.exist();
                 expect(err).to.be.instanceOf(Error);
-                done();
-            });
+            }
         });
 
-        it('connects with authentication', (done) => {
+        it('connects with authentication', async () => {
 
             const options = {
                 uri: 'mongodb://tester:secret@127.0.0.1:27017/?maxPoolSize=5',
                 partition: 'unit-testing'
             };
             const mongo = new Mongo(options);
 
-            mongo.start((err) => {
-
+            try {
+                await mongo.start();
+            }
+            catch (err) {
                 expect(err).to.not.exist();
-                done();
-            });
+            }
         });
 
-        it('sets isReady to true when the connection succeeds', (done) => {
+        it('sets isReady to true when the connection succeeds', async () => {
 
             const options = {
                 uri: 'mongodb://127.0.0.1:27017/?maxPoolSize=5',
                 partition: 'unit-testing'
             };
             const mongo = new Mongo(options);
 
-            mongo.start((err) => {
-
+            try {
+                await mongo.start();
+            }
+            catch (err) {
                 expect(err).to.not.exist();
                 expect(mongo.isReady()).to.be.true();
```

Comment:
You are testing isReady on place where it never going to be called. This test could look something like this:
```
await mongo.start()
expect(mongo.isReady()).to.be.true();
```
If the mongo.start would reject instead of resolve, this test would end up with error, so no need always for explicit catch.

Reply:
Good catch! 

You’re right, the test fails `mongo.start()` rejects and therefore the test fails.